components/BannerManager/Liquidation/useLiquidationOptimismSubgraph.ts

```diff
@@ -52,30 +46,41 @@ export async function fetchLiquidationInfo(
 		}),
 	});
 
-	const { data, errors } = await body.json();
+	const {
+		errors,
+		data,
+	}: {
+		errors?: Error[];
+		data: { stakerEntity: StakerEntity };
+	} = await body.json();
+
 	if (errors?.[0]) {
 		throw new Error(errors?.[0]?.message || 'Unknown server error');
 	}
 	return data?.stakerEntity;
 }
 
-export function useLiquidationOptimismInfo(queryOptions?: UseQueryOptions) {
+export function useLiquidationOptimismInfo(queryOptions?: UseQueryOptions<StakerEntity>) {
```

Comment:
Yea its a bit weird. As I wrote in that other comment it is good to infer if you dont let consumer pass in the option. But if you pass in options that you have typed it will use those options to "create" the return type. (I think)